composer.json

```diff
@@ -14,7 +14,13 @@
     },
     "autoload": {
         "psr-4": {
-          "Google\\GAX\\":"src/"
+          "Google\\GAX\\":"src/",
+          "google\\protobuf\\":"src/generated/",
+          "google\\api\\":"src/generated/",
+          "google\\rpc\\":"src/generated/",
+          "google\\type\\":"src/generated/",
+          "google\\longrunning\\":"src/generated/",
+          "google\\logging\\":"src/generated/"
```

Comment:
Does this actually work with autoloading? The filenames aren't named the same as the classes as per PSR-4.